File: patientdataprocessor-backend-microservice/authentication-service/src/test/java/com/cts/authenticationservice/model/Results.java
package com.cts.authenticationservice.model;

import java.util.Date;
import java.util.Locale;

import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import com.fasterxml.jackson.annotation.JsonFormat;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class Results {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;
	
	private Long contactNumber;
	
	@JsonFormat(pattern = "yyyy/MM/dd hh:mm:ss ")
	private Locale date;
	
	private double whiteBloodCells;
	
	private double redBloodCells;
	
	private double hemoglobin;
	
	private double hemotocrit;
	
	private double meanCellVolume;
	
	private double meanCellHemoglobin;
	
	private int  platelets;
	
	private double reticulocyte;
	
}

File: Temp/frontend/latest/patientdataprocessor-frontend/src/app/model/results.ts
export class Results {
    constructor(
        public id: number,
        public contactNumber: number,

        public whiteBloodCells: number,

        public redBloodCells: number,

        public hemoglobin: number,

        public hemotocrit: number,

        public meanCellVolume: number,

        public meanCellHemoglobin: number,

        public platelets: number,

        public reticulocyte: number,

        public date: DateConstructor
    ) { }
}